README.md

```diff
@@ -4,45 +4,51 @@
 
 A curated list of [awesome](https://github.com/sindresorhus/awesome) Cedar related tools and articles. Amazon does not endorse or support the hyperlinked projects, and expressly disclaims all warranties, whether express, implied or statutory, regarding these projects. Use them “as is” and “as available” at your own discretion.
 
-## Offical 
-  - [Cedar Project](https://www.cedarpolicy.com/en)  
-  - [Cedar Docs](https://docs.cedarpolicy.com/)
-  - [Cedar Playground](https://www.cedarpolicy.com/en/playground)
-  - [Cedar Git](https://github.com/cedar-policy)
-  - [Cedar Community](https://io.permit.io/07ATaP)
+## Offical
+
+- [Cedar Project](https://www.cedarpolicy.com/en)  
+- [Cedar Docs](https://docs.cedarpolicy.com/)
+- [Cedar Playground](https://www.cedarpolicy.com/en/playground)
+- [Cedar Git](https://github.com/cedar-policy)
+- [Cedar Community](https://io.permit.io/07ATaP)
 
 ## Deploy and Run Cedar
-  - [Amazon Verified Permissions](https://aws.amazon.com/verified-permissions/)
-  - [Cedar Agent](https://github.com/permitio/cedar-agent)
-  - [Permit.io](https://www.permit.io/) 
 
+- [Amazon Verified Permissions](https://aws.amazon.com/verified-permissions/)
+- [Cedar Agent](https://github.com/permitio/cedar-agent)
+- [Permit.io](https://www.permit.io/)
 
 ## Language and Platform Integrations
-  - Go bindings [WASM based](https://github.com/Joffref/cedar) 
-  - Go bindings [CGO based](https://github.com/iann0036/cedargo)
-  - [Python bindings (cedarpy)](https://github.com/k9securityio/cedar-py)
-  - [Highlight.js syntax](https://github.com/jasmaa/highlightjs-cedar)
+
+- Go bindings [WASM based](https://github.com/Joffref/cedar)
+- Go bindings [CGO based](https://github.com/iann0036/cedargo)
+- [Python bindings (cedarpy)](https://github.com/k9securityio/cedar-py)
+- [Highlight.js syntax](https://github.com/jasmaa/highlightjs-cedar)
+
+## IDE Extensions
+
+- [VSCode](https://marketplace.visualstudio.com/items?itemName=cedar-policy.vscode-cedar)
```

Comment:
can we link to both the Marketplace link and the code repo?

Reply:
Sure, added. Will render as shown below - does it look ok?

- VSCode ([GitHub](https://github.com/cedar-policy/vscode-cedar), [Marketplace](https://marketplace.visualstudio.com/items?itemName=cedar-policy.vscode-cedar))

Reply:
looks great